packages/genesys-spark-components/src/components/beta/gux-month-picker/gux-month-calendar/gux-month-calendar.tsx

```diff
@@ -94,6 +107,36 @@ export class GuxMonthCalendar {
     });
   }
 
+  @Listen('keydown')
+  onKeyDown(event: KeyboardEvent): void {
+    switch (event.key) {
+      case 'Tab':
+        // Manually move focus to the selected month if the focus is currently on the next year arrow in the header
+        if (!event.shiftKey && this.nextYearElement.matches(':focus-visible')) {
+          const monthElement: HTMLButtonElement =
+            this.root.shadowRoot.querySelector(
+              `gux-month-list-item[value="${this.previewValue}"]`
+            );
+          if (monthElement) {
+            monthElement.focus();
+          }
+          event.preventDefault();
+        }
+        break;
+    }
+  }
+
+  @Listen('focusout')
+  onFocusout(event: FocusEvent): void {
+    const focusIsOutsideComponent = !this.root.contains(
+      event.relatedTarget as Node
+    );
+    if (focusIsOutsideComponent) {
+      this.expanded = false;
+      this.closeCalendar.emit();
```

Comment:
@katie-bobbe-genesys This is how i notify the parent that the calendar is closed. I do this so that the parent can be told of the state of the calendar's visibility since the parent tracks that itself.